### DISCUSSION TITLE
Docker Compose Configuration with Persistent Storage for Deploying Microcks on Azure Web App

### DISCUSSION THREAD

**Discussion by mohammad-anbari-maersk** [microcks/microcks#1238] [Q&A]
Hello,

Thank you for your awesome product.

I am new to Microcks and very interested in installing and setting it up. I want to deploy Microcks as a container on Azure Web App using a Docker Compose file. Is there a Docker Compose file available from your product that includes a persistent storage option?

Currently, I am using a simple Docker Compose file which starts Microcks, but upon restarting the container, all imported APIs are lost.

Thank you!

**Comment by lbroudoux** [comment]
Hello,

Yes, we provide a Docker Compose installation method. You can check the documentation here: https://microcks.io/documentation/guides/installation/docker-compose/ to get information on different variants available.

As the default docker-compose file (see https://github.com/microcks/microcks/blob/master/install/docker-compose/docker-compose.yml#L8) is mounting a volume to persist MongoDB data, you should be able to adapt this configuration to use a persistent disk in Azure.

Let us know how things are going!
